value/pom.xml

```diff
@@ -139,33 +143,34 @@
 	    </goals>
 	    <configuration>
 	      <minimizeJar>true</minimizeJar>
-              <filters>
-                <filter>
-                   <artifact>org.apache.velocity:*</artifact>
```

Comment:
I'm confused. Does this mean that people will have to express an explicit dependency on this if they want to use AutoValue?